ansible/roles/zookeeper-upgrade/templates/zoo.cfg.j2

```diff
@@ -8,6 +8,6 @@ maxClientCnxns={{ maxclinetconnection_limit }}
 
 
 {% for host in groups['zookeeper'] %}
-server.{{hostvars[host]['server_id']}}={{hostvars[host]['ansible_eth0']['ipv4']['address']}}:2888:3888
+server.{{hostvars[host]['server_id']}}={{hostvars[host].ansible_default_ipv4.address }}:2888:3888
```

Comment:
@anandp504 Can you please take the copy of this file from release-1.15? The one in 1.15 is correct so this change should not get merged.